src/model.rs

```diff
@@ -145,6 +159,13 @@ impl ClusterState {
         self.node_states.get(node_id)
     }
 
+    // TODO: Just a demo.
```

Comment:
Currently there is no way to consume this API. There is no way to access all nodes from the outside at all (just a specific node, but how would a consumer know about it to begin with?). I'd assume the information one would care about externally is the `node_id` and k/v of all nodes (so basically everything). In that case one could probably just expose this method publicly through `ScuttleButt`? Using the `node_id`s then the consumer can access the k/v of all nodes, alternatively this method could expose k/v and node_id at the same time, since it wouldn't really be any additional effort anyway.

Reply:
Yes I think it is a good idea to have this method.

I recommend &str instead of &String however.
(Granted, it does not matter as much as a return value)